app/code/community/Sendsmaily/Sync/controllers/Newsletter/SubscriberController.php

```diff
@@ -28,8 +28,9 @@ class Sendsmaily_Sync_Newsletter_SubscriberController extends Mage_Adminhtml_New
    *
    * @return void
    */
-  public function exportToSendsmailyAction() {
-    if(Mage::getStoreConfig('newsletter/sendsmaily/active') == false){
+  public function exportToSendsmailyAction()
```

Comment:
I left in the old functionality. I don't know if that is required. Maybe it's convenient for customers who want to update their unsubscribers before launching a campaign or something.

Reply:
Let's keep it for now. Maybe somebody uses it instead of CRON.